src/targets/gpu/include/migraphx/gpu/hip.hpp

```diff
@@ -144,17 +144,16 @@ struct hip_copy_from_gpu
         check_shapes{inputs, *this}.has(1, 2);
         return inputs.at(0);
     }
-    argument
-    compute(context& ctx, const shape& output_shape, const std::vector<argument>& args) const
+    argument compute(context& ctx, const shape&, const std::vector<argument>& args) const
     {
         if(args.size() == 1)
         {
-            argument result = allocate_gpu(output_shape, true);
+            argument result = allocate_gpu(args.front().get_shape(), true);
```

Comment:
`output_shape` should be the same otherwise we have a bug in computing the shape.